src/dmd/doc.d

```diff
@@ -1823,6 +1823,18 @@ extern (C++) bool isDitto(const(char)* comment)
     return false;
 }
 
+bool isWhitespace(const(char) c)
+{
+// TODO: unicode whitespace: Zs.
+    return c == ' ' || c == '\t' || c == '\r' || c == '\n';
+}
+
+bool isPunctuation(const(char) c)
+{
+// TODO: unicode punctuation: Pc, Pd, Pe, Pf, Pi, Po, or Ps.
+    return c == '!' || c == '"' || c == '#' || c == '$' || c == '%' || c == '&' || c == '\'' || c == '(' || c == ')' || c == '*' || c == '+' || c == ',' || c == '-' || c == '.' || c == '/' || c == ':' || c == ';' || c == '<' || c == '=' || c == '>' || c == '?' || c == '@' || c == '[' || c == '\\' || c == ']' || c == '^' || c == '_' || c == '`' || c == '{' || c == '|' || c == '}' || c == '~';
```

Comment:
Please break this line into multiple lines. Ideally a line should not exceed 80 columns.